1-exercises/A-this/exercise-1.js

```diff
@@ -15,7 +15,9 @@ let person = {
     name: "Adrian",
     age: 32,
     currentAddress: "London",
-    celebrateBirthday: /* todo */
+    celebrateBirthday: function(){
+        return this.age+1
+    }
 };
```

Comment:
This is a very nice and functional feature, however, you no longer have to include: and write functions when writing methods (functions inside objects).

changeAddress (newAddress) {
this.currentAddress = newAddress;    
}

This gives you the same result.